### PR TITLE
Add return stmt if not present and fix debugger tests

### PR DESCRIPTION
**Pull request by lankavitharana**

### CODE REVIEW

File: modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java [hunk 1]
```diff
@@ -442,7 +442,10 @@ public void visit(BLangBlockStmt blockNode) {
         SymbolEnv blockEnv = SymbolEnv.createBlockEnv(blockNode, this.env);
 
         for (BLangStatement stmt : blockNode.stmts) {
-            addLineNumberInfo(stmt);
+            if (stmt.getKind() != NodeKind.TRY && stmt.getKind() != NodeKind.CATCH
```
Comment (hasithaa) [inline modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java]: what if there transaction ?

Comment (lankavitharana) [reply modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java]: Transaction scenarios are not tested with debugger yet, will test them and improve this transaction scenarios later

File: modules/ballerina-core/src/main/java/org/ballerinalang/bre/Context.java [hunk 2]
```diff
@@ -50,6 +50,7 @@
     private BallerinaTransactionManager ballerinaTransactionManager;
     private DebugInfoHolder debugInfoHolder;
     private boolean debugEnabled = false;
+    private boolean nonBlockingActionCall = false;
```
Comment (hasithaa) [inline modules/ballerina-core/src/main/java/org/ballerinalang/bre/Context.java]: Please add this field in WorkerContext's populateContextPropsFromParent() method.

File: modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java (hunk 1 above)
Comment (lankavitharana) [reply modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java]: created a issue to track this https://github.com/ballerinalang/ballerina/issues/3591

**Pull request merged**